internal/config/profile.go

```diff
@@ -52,9 +52,9 @@ const (
 	service                      = "service"
 	publicAPIKey                 = "public_api_key"
 	privateAPIKey                = "private_api_key"
-	accessToken                  = "access_token"
-	refreshToken                 = "refresh_token"
-	opsManagerURL                = "ops_manager_url"
+	AccessTokenField             = "access_token"
```

Comment:
[q] What are these not camelCase (with a lowercase starting char) like the rest of the entries in the file?

Reply:
Public access